cryodrgn/dataset.py

```diff
@@ -16,7 +16,7 @@
 log = utils.log
 
 
-def load_particles(mrcs_txt_star, lazy=False, datadir=None):
+def load_particles(mrcs_txt_star: str, lazy: bool = False, datadir: bool = None):
```

Comment:
One place where `pyright` fails - cannot have default `None` if the type is `bool`.